src/Todoist.Net/Todoist.Net.csproj

```diff
@@ -25,6 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="8.0.0" />
```

Comment:
I recommend including the dependency only for the .NET Core version of the library. There's no need to include it for the full framework. Additionally, we should consider lowering the version to the latest LTS.

Reply:
Done 👍